FirebaseInAppMessaging/Sources/Data/FIRIAMMessageDefinition.m

```diff
@@ -36,23 +36,39 @@ @implementation FIRIAMMessageDefinition
 - (instancetype)initWithRenderData:(FIRIAMMessageRenderData *)renderData
                          startTime:(NSTimeInterval)startTime
                            endTime:(NSTimeInterval)endTime
-                 triggerDefinition:(NSArray<FIRIAMDisplayTriggerDefinition *> *)renderTriggers {
+                 triggerDefinition:(NSArray<FIRIAMDisplayTriggerDefinition *> *)renderTriggers
+                           appData:(nullable NSDictionary *)appData
+                     isTestMessage:(BOOL)isTestMessage {
   if (self = [super init]) {
     _renderData = renderData;
     _renderTriggers = renderTriggers;
     _startTime = startTime;
     _endTime = endTime;
-    _isTestMessage = NO;
+    _isTestMessage = isTestMessage;
+    _appData = appData;
```

Comment:
`_appData = [appData copy];` is more preferred to avoid _appData being NSMutableDictionary

Reply:
Thanks for the comments David. I'll make those changes now :)